assets/multiqc_config.yaml

```diff
@@ -1,11 +1,11 @@
 report_comment: >
```

Comment:
You shouldn't need this file anymore since you have the `assets/multiqc_config.yml`

Reply:
I have deleted this in the PR from the `variant_calling` branch to `dev`:
https://github.com/nf-core/nanoseq/pull/185
Thank you!!